src/main/java/uk/gov/hmcts/darts/common/enums/SecurityGroupEnum.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.darts.common.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum SecurityGroupEnum {
```

Comment:
pity we need to introduce another enum when we have already `SecurityRoleEnum`. Perhaps there's a way to reuse that one?

Reply:
Role and group are different things. Not sure there is a way to reuse role. I explicitly need to check the group name.